server/repository/src/db_diesel/changelog/test.rs

```diff
@@ -39,6 +39,7 @@ async fn test_changelog() {
             r.table_name = ChangelogTableName::Location;
             r.record_id = mock_location_1().id;
             r.row_action = RowActionType::Upsert;
+            r.store_id = Some(mock_location_1().store_id.clone());
```

Comment:
`store_id` wasn't be set in location upsert trigger. I've added it now, feels like the right thing to do, but let me know if not...